inference-engine/ie_bridges/java/samples/CMakeLists.txt

```diff
@@ -1,10 +1,6 @@
 # Copyright (C) 2020 Intel Corporation
 
-find_package(OpenCV) 
-if(NOT OpenCV_FOUND)
-    message(WARNING ".jar file wasn't found for OpenCV. Java samples won't be build.")
-    return()
-endif()
+find_package(OpenCV)
```

Comment:
why OpenCV is mandatory?

Reply:
This file seems strange as for me:
```cmake
find_package(OpenCV)
if(EXISTS "${java_opencv_src}")
    ...
else()
    message(WARNING ".jar file wasn't found for OpenCV.\n Java samples, except Benchmark Applicaton, won't be build.")
    add_subdirectory(benchmark_app)
endif()
```

I think that we should build `benchmark_app` anyway